internal/platform/data/rules_test.go

```diff
@@ -52,19 +54,19 @@ func TestRules(t *testing.T) {
 			Output: []RuleOutput{
 				{
 					"type":  "direct-translation",
-					"value": "{{(index .Words 2).Lemma}}",
+					"value": "{{( .Words 2).Lemma}}",
```

Comment:
I am a little bit confuse, in our documentation we have the output like:
 "value": "{{Word2.Lemma}}",

but here we have 
"value": "{{( .Words 2).Lemma}}",

which does it the final?

Reply:
It is a typo thanks for find it, could you fix it in your next PR?

Reply:
I will use the format 

"value": "{{ .Word2.Lemma }}"